### DISCUSSION TITLE
Did I just see...?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#148] [General]
I could swear I saw "SNOOPY" flash at the bottom of the console (milliseconds), but the only place those chars are even in DOG are in the NAME line, but never used.

Does DSD display the NAME line MICRO?

**Comment by KC5VDJ** [comment]
Just out of curiosity, also, when exactly was "A Boy Named Charlie Brown" on the screen?  Dates.  Relevance (local even).

**Comment by drboone** [comment]
HUSTLER displays Mickus Mouse briefly during startup.

**Reply by KC5VDJ** [reply]
Yeah, and I think Kronos as well...iirc.  need to get that going again.  Some PP code I can use for testing is for it.
